utils.py

```diff
@@ -19,7 +19,8 @@ def send_email(logger, subject, msg, recipient=None):
     template_path = "templates/email_template.html"
     with open(os.path.join(os.path.dirname(__file__), template_path), 'r') as input_file:
         html = input_file.read()
-    logo_src = "http://" + app.config["SERVER_NAME"] + "/static/assets/img/logo.png"
+    logo_src = app.config["LOGO_EMAIL_SRC"]
+    #logo_src = "http://" + app.config["SERVER_NAME"] + "/static/assets/img/logo.png"
```

Comment:
remove dead (commented) code

Reply:
removed the commented line and pushed again.